### PR TITLE
Allow select option quick to select among more than 5 values

### PR DESCRIPTION
**Pull request by edo9300**

### CODE REVIEW

**Comment by edo9300** [comment]
If tehre are more than 5 options to choose from, and all those options' texts can enter into the buttons a scrollbar will be added to allow you to better choose among them 
image

**Comment by mercury233** [comment]
I didn't test it but shouldn't we reset `scrOption` in `ShowSelectOption` ?

**Comment by edo9300** [comment]
wait..., damn github desktop.. it didn't push all teh files

File: gframe/client_field.cpp [hunk 1]
```diff
@@ -594,26 +594,34 @@ void ClientField::ShowSelectOption(int select_hint) {
...
-		if(dtxt.Width > 310) {
+	for(auto option : select_options) {
+		if(mainGame->guiFont->getDimension(dataManager.GetDesc(option)).Width > 310) {
 			quickmode = false;
 			break;
 		}
+	}
+	for(int i = 0; (i < count) && quickmode; i++) {
+		const wchar_t* option = dataManager.GetDesc(select_options[i]);
 		mainGame->btnOption[i]->setText(option);
```
Comment (VanillaSalt) [inline gframe/client_field.cpp]: out of bound if count > 5

**Comment by purerosefallen** [comment]
does it work fine with resize?

**Comment by edo9300** [comment]
@purerosefallen i remember you that that still hasn't been merged...

**Comment by purerosefallen** [comment]
@edo9300 but did you change anything else on EDOPro?

**Pull request merged**